demangle-mode.el

```diff
@@ -60,6 +60,21 @@
   "Automatically demangle C++ symbols found in buffers."
   :group 'tools)
 
+(defcustom demangle-cxxfilt-executable "c++filt"
```

Comment:
Existing Emacs symbols that involve C++ generally use `c++` in the symbol name instead of `cxx`, such as `c++-font-lock-keywords`, `c++-mode`, etc. Also, the default filtering executable is `c++filt`, not `cxxfilt`. So let’s use `++` here too. Please rename the new symbols as follows:

- `demange-cxxfilt-executable` → `demangle-c++filt-executable`
- `demangle-cxxfilt-args` → `demangle-c++filt-args`
- `demangle--cxxfilt-command` → `demangle--c++filt-command`